Merging changes from GraalPy master and updating docs (#34)
graalvm/graalpy.github.io: docs/graalpy-embedded/reference/interop/from_python_to_other_langs.md

> ## Exporting Python objects to other languages

Reviewer: ```suggestion
## Exporting Python Objects to other Languages
```

> The polyglot module is also useful to expose Python objects to both other GraalVM languages as well as JVM languages.

Reviewer: ```suggestion
The `polyglot` module can be used to expose Python objects to other GraalVM languages as well as to other JVM languages.
```
